staging/src/k8s.io/apiextensions-apiserver/Godeps/Godeps.json

```diff
@@ -174,6 +174,10 @@
 			"ImportPath": "github.com/golang/protobuf/ptypes/timestamp",
 			"Rev": "4bd1920723d7b7c925de087aa32e2187708897f7"
 		},
+		{
+			"ImportPath": "github.com/google/btree",
+			"Rev": "7d79101e329e5a3adf994758c578dab82b90c017"
```

Comment:
Are these changes correct?

Reply:
This is odd. Is it the only one added? Or anything changed? Maybe godep behaviour is slightly different, or we had invalid Godeps.json? Does our CI test this?